integration/conftest.py

```diff
@@ -310,12 +310,13 @@ def storage_nodes(reactor, temp_dir, introducer, introducer_furl, flog_gatherer,
     # start all 5 nodes in parallel
     for x in range(5):
         name = 'node{}'.format(x)
+        web_port=  9990 + x
         # tub_port = 9900 + x
         nodes.append(
             pytest_twisted.blockon(
                 _create_node(
                     reactor, request, temp_dir, introducer_furl, flog_gatherer, name,
-                    web_port=None, storage=True,
+                    web_port=str(web_port), storage=True,
```

Comment:
In fact the `web_port` parameter to `_create_node` should probably just be deleted.  Or at least changed so it's not possible to pre-assign an integer port number.

Reply:
(Taking discussion of port-numbers to `#tahoe-lafs` .. but also will note that there are other pre-assigned port-numbers in the integration tests)

Reply:
I don't know that we came to a conclusion in `#tahoe-lafs` but given:

 - we already have static ports in integration tests; 
 - and there's no better solution that works with tahoe;

...I don't see why we should halt this PR in order to come up with a solution. There won't be any conflicts within the integration tests. There *could* be conflicts on the host system (but that is true if we use other, more-random methods of assigning ports). We can't run the tests in parallel (which is true either way, regardless of this PR).

(Note: trying to file a ticket but SpamBayes doesn't like my words .. I think this is best addressed as a separate ticket because this isn't the first hard-coded port, nor first hard-coded web-port).

Reply:
okay, convinced trac to accept a ticket for this: https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3249#ticket